metricbeat/module/elasticsearch/cluster_stats/data.go

```diff
@@ -50,30 +159,259 @@ var (
 			}),
 		}),
 	}
+
+	stackSchema = s.Schema{
+		"apm": c.Ifc("apm"),
+		"xpack": c.Dict("xpack", s.Schema{
+			"rollup":               c.Ifc("rollup"),
+			"logstash":             c.Ifc("logstash"),
+			"transform":            c.Ifc("transform"),
+			"security":             c.Ifc("security"),
+			"data_streams":         c.Ifc("data_streams"),
+			"monitoring":           c.Ifc("monitoring"),
+			"graph":                c.Ifc("graph"),
+			"voting_only":          c.Ifc("voting_only"),
+			"slm":                  c.Ifc("slm"),
+			"frozen_indices":       c.Ifc("frozen_indices"),
+			"spatial":              c.Ifc("spatial"),
+			"searchable_snapshots": c.Ifc("searchable_snapshots"),
+			"ccr":                  c.Ifc("ccr"),
+			"vectors":              c.Ifc("vectors"),
+			"ilm": c.Dict("ilm", s.Schema{
+				"policy": s.Object{
+					"total": c.Int("policy_count"),
+					"stats": c.Ifc("policy_stats"),
+				},
+			}),
+			//"watcher": c.Ifc("watcher"),
```

Comment:
Why is this field commented out?

Reply:
I forgot to add a comment for double check it's not required (I was hoping it wasn't). If Chris hasn't complain I guess it's not so I have removed it :smile:

Reply:
Is this data contained in the `stack_stats` field?

Reply:
It wasn't included when you first tested it, @chrisronline I guess it's safe to remove it